web-client/integration-tests/messagesJourney.test.js

```diff
@@ -110,4 +111,68 @@ describe('messages journey', () => {
   });
   petitionsClerkVerifiesCompletedMessageNotInInbox(test);
   petitionsClerkVerifiesCompletedMessageNotInSection(test);
+
+  loginAs(test, 'petitionsclerk@example.com');
+  it('attaches a document to a message with a very long title, which is truncated in the subject', async () => {
+    await test.runSequence('gotoCaseDetailSequence', {
+      docketNumber: test.docketNumber,
+    });
+
+    await test.runSequence('openCreateMessageModalSequence');
+
+    await test.runSequence('updateSectionInCreateMessageModalSequence', {
+      key: 'toSection',
+      value: PETITIONS_SECTION,
+    });
+
+    await test.runSequence('updateModalFormValueSequence', {
+      key: 'toUserId',
+      value: '4805d1ab-18d0-43ec-bafb-654e83405416', //petitionsclerk1
+    });
+
+    const currentDocketEntries = test.getState('caseDetail.docketEntries');
+    const longDocumentTitle = getTextByCount(255);
+
+    const docketEntryWithLongTitle = {
+      addToCoversheet: false,
+      createdAt: '2021-04-19T19:22:08.389Z',
+      docketEntryId: '999cd272-da92-4e31-bedc-28cdad2e08b0',
+      docketNumber: '304-21',
+      documentTitle: longDocumentTitle,
+      documentType: longDocumentTitle,
```

Comment:
Just making sure you want `documentType` to be `longDocumentTitle`?

Reply:
Yeah, not an issue here, as they can sometimes be the same (and in fact were in some other test data in these tests).